### ISSUE TITLE
got exception : <class 'IndexError'> : list index out of range

### ISSUE THREAD

**Issue by em1le** [Kozea/WeasyPrint#1009]
Hi there, i just got a bug.

Here is the stack trace :

### weasyprint/layout/inlines.py in skip_first_whitespace at line 210
```python
        return (index, None) if index else None
    if isinstance(box, (boxes.LineBox, boxes.InlineBox)):
        if index == 0 and not box.children:
            return None
        result = skip_first_whitespace(box.children[index], next_skip_stack)
        if result == 'continue':
            index += 1
            if index >= len(box.children):
                return 'continue'
            result = skip_first_whitespace(box.children[index], None)
```
```
box  | <InlineBox br>
index | 8
next_skip_stack | None
skip_stack | [8, None]
```

I do not have any clue where does that come from.

If i may help with more details, let me know.

```

**Comment by em1le** [comment]
Quick fix found in #783 by @knixeur in its first comment

**Comment by knixeur** [comment]
Glad you sorted it out, however that was a quick fix to keep working, you should try using the version from master in this repo to see if the real fix is already included. Otherwise if you could provide a simple sample to reproduce it it would be great.
